### PR TITLE
feat(odata-service-inquirer): 37197 responsive table info message

### PR DESCRIPTION
**Pull request by korotkovao**
Related to internal issue 37197

Display an info message (blue info icon) when the Responsive Table is selected:

Info text: ℹ️ Only use the responsive table if the total number of items in the table doesn't exceed 200.

Screenshot 2026-02-26 at 13 11 54

### CODE REVIEW

File: packages/odata-service-inquirer/src/prompts/edmx/questions.ts [hunk 1]
```diff
@@ -336,13 +336,21 @@ function getTableLayoutQuestions(
...
+            additionalMessages: (selectedTableType: TableType) => {
+                // Show responsive table info whenever ResponsiveTable is selected
+                if (selectedTableType === 'ResponsiveTable') {
+                    return {
+                        message: t('prompts.tableType.responsiveTableInfo'),
+                        severity: Severity.information
+                    };
+                }
+                // Show default type info when the default is selected
+                if (defaultTableType === 'AnalyticalTable' && selectedTableType === 'AnalyticalTable') {
```
Comment (IainSAP) [inline packages/odata-service-inquirer/src/prompts/edmx/questions.ts:348]: Do we need to update a test to cover these new conditions also? 
`if (defaultTableType === 'AnalyticalTable' && selectedTableType === 'AnalyticalTable')`
`if (defaultTableType === 'TreeTable' && selectedTableType === 'TreeTable') {`

Comment (korotkovao) [reply packages/odata-service-inquirer/src/prompts/edmx/questions.ts:348]: Thanks @IainSAP it should be covered now

these conditions are covered by existing tests in questions.test.ts:

- `defaultTableType === 'AnalyticalTable' && selectedTableType === 'AnalyticalTable'` (tests `additionalMessages` returns `analyticalTableDefault` message when default is `AnalyticalTable`)
- `defaultTableType === 'TreeTable' && selectedTableType` === 'TreeTable'  (tests `additionalMessages'TreeTable' returns `treeTableDefault` message when default is `TreeTable`)
- `selectedTableType === 'ResponsiveTable'` (tests `responsiveTableInfo` message)
- Edge case when default differs from selection (tests `additionalMessages('GridTable')` returns `undefined` when default is `AnalyticalTable` but user selects `GridTable`)

File: .changeset/social-clouds-smell.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/odata-service-inquirer': minor
+---
+
+Display an info message when the Responsive Table is selected.
```
Comment (JannaLisa) [inline .changeset/social-clouds-smell.md]:
```suggestion
Display an info message when the responsive table is selected.
```
I wouldn't capitalize, since this isn't a proper noun.

**Comment by korotkovao** [comment]
Thanks @IainSAP @JannaLisa
